tests/testthat/test_pyCall.R

```diff
@@ -0,0 +1,19 @@
+context("test pyCall returns data.frame")
+
+test_that("pyCall() returns r data.frame for pandas DataFrame", {
+  # import pandas in python and set pyOptions to use pandas
+  source(file.path(getwd(), "tests", "testthat", "install_pandas.R"))
+
+  # creating test data
+  df <- data.frame(c(1,2,3), c(T,F,F))
+  expect_equal(class(df), "data.frame")
+  
+  pySet("df", df)
+  expect_output(pyExecp("type(df)"), "pandas.core.frame.DataFrame")
+
+  # define a test function in python
+  pyExec("return_df = lambda: df")
+
+  df2 <- pyCall("return_df")
+  expect_equal(class(df2), "data.frame")
```

Comment:
should clean up python pandas after the test (or install on a different location and ignore this location)